### PR TITLE
test: fix testNonAdminUser test

### PR DESCRIPTION
**Pull request by jkozol**

### CODE REVIEW

**Comment by jkozol** [comment]
testNonAdminUser passes so I would consider the failed tests passes since they will be fixed in the create-wizard test PR #1479

File: test/verify/check-service [hunk 1]
```diff
@@ -112,12 +112,12 @@ class TestService(composerlib.ComposerCase):
         b.wait_visible("button:contains('Start')")
 
         # disable this test due to https://github.com/osbuild/cockpit-composer/issues/941
```
Comment (lucasgarfield) [inline test/verify/check-service:114]: I realize that the issue is still open but I imagine this comment could be really confusing if left in place in a few years when we've all forgotten about the unification...

Comment (jkozol) [reply test/verify/check-service:114]: nice catch.

**Review by lucasgarfield — APPROVED** [review]
LGTM!

**Pull request merged**